Client/TabManagement/Legacy/LegacyTabManager.swift

```diff
@@ -179,13 +179,6 @@ class LegacyTabManager: NSObject, FeatureFlaggable, TabManager, TabEventHandler
                                                object: nil)
         // Ecosia: Cookie and settings observing
         configuration.websiteDataStore.httpCookieStore.add(self)
-
-        searchSettingsObserver = NotificationCenter.default
-            .publisher(for: .searchSettingsChanged)
-            .sink() { [privateConfiguration, configuration] _ in
-                configuration.websiteDataStore.httpCookieStore.setCookie(Cookie.makeStandardCookie())
-                privateConfiguration.websiteDataStore.httpCookieStore.setCookie(Cookie.makeIncognitoCookie())
-            }
```

Comment:
I guess this is still relevant for keeping our search settings updated with web, why were they removed?